module_book.rb

```diff
@@ -0,0 +1,20 @@
+require './book'
```

Comment:
- [ ] Make sure to use `require_relative 'file'` every time you are importing relative files, this is a best practice on Ruby. [You can read more about it here.](https://rubystyle.guide/#use-require_relative-whenever-possible)

Reply:
Hey @D4nielJ 
Thank you.